style.css

```diff
@@ -1,5 +1,6 @@
 @import "stylesheets/Components/alert.css";
-
+@import "stylesheets/Components/avatar.css";
+@import "stylesheets/Components/button.css";
```

Comment:
Mention the component elements below the CSS, so that other general files do not overshadow your code

Reply:
done . do tell if anything else is required to be done

Reply:
Looks good